python/doc/examples/probabilistic_modeling/random_vectors/plot_ratioOfUniforms_exploitation.py

```diff
@@ -0,0 +1,337 @@
+"""
+Use the Ratio of Uniforms algorithm to sample a distribution
+============================================================
+"""
+
```

Comment:
imho we should stick to lowercase file names for examples

Reply:
done